distribution/rpm/build.gradle

```diff
@@ -18,6 +18,13 @@
  */
 
 task buildRpm(type: Rpm) {
+  signingKeyId = project.hasProperty('signing.keyId') ? project.getProperty('signing.keyId') : System.getProperty('signing.keyId', 'D88E42B4')
```

Comment:
Can't all of this be set in the ospackage section of distribution/build.gradle, so it applies to both deb and rpm?